src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/MergedCalController.java

```diff
@@ -34,22 +33,28 @@
 
     @PostMapping(value = "/calendar")
     public ResponseEntity postMergedCal(@RequestHeader("X-Client-Token") String clientToken,
-        @RequestBody MergedCalendarFromAndroid mergedCalendarFromAndroid) {
+                                        @RequestBody MergedCalendarFromAndroid mergedCalendarFromAndroid) {
         MergedCalendar mergedCalendar = new MergedCalendar();
         mergedCalendar.setOutputCalendarId(mergedCalendarFromAndroid.getOutputCalendarId());
         String[] inputStrings = mergedCalendarFromAndroid.getInputCalendarIds();
         mergedCalendar.setCalendarIds(mergedCalendar.getCalendarIds(inputStrings));
         mergedCalendar
-            .setUserName(userModelRepository.findByClientToken(clientToken).getUserEmail());
+                .setUsername(userModelRepository.findByClientToken(clientToken).getUserEmail());
         mergedCalendar
-            .setOutputAccount(userModelRepository.findByClientToken(clientToken).getUserEmail());
+                .setOutputAccount(userModelRepository.findByClientToken(clientToken).getUserEmail());
         mergedCalendarRepository.save(mergedCalendar);
         return new ResponseEntity(HttpStatus.OK);
     }
 
     @GetMapping(value = "/calendar")
-    public MergedCalendarResponse getMergedCalList() {
-        return new MergedCalendarResponse();
-    }
+    public ResponseEntity getMergedCalList(@RequestHeader("X-Client-Token") String clientToken, HttpServletRequest request) throws IOException {
+        UserModel userModel;
+        if (!request.getHeader("X-Client-Token").equals("")) {
```

Comment:
@RedRiot extract these magic strings into a separate class that contains only string constants